### DISCUSSION TITLE
Middleware groups

### DISCUSSION THREAD

**Discussion by inmanturbo** [spatie/laravel-route-attributes#42] [Q&A]
is it possible to use middleware groups?

**Comment by Blitheness** [comment]
Yes, you can add a middleware group such as "web", in the same way that you'd add a single middleware item to a route.

> Middleware groups may be assigned to routes and controller actions using the same syntax as individual middleware.

https://laravel.com/docs/9.x/middleware#middleware-groups

**Reply by msacharewicz** [reply]
Ref.: [#37#discussioncomment-4888700](https://github.com/spatie/laravel-route-attributes/discussions/37#discussioncomment-4888700), use the config `'directories'` option to configure your groups app-wide.
